actors/infinispan-cluster/src/main/java/cloud/orbit/actors/cluster/JGroupsClusterPeer.java

```diff
@@ -49,19 +49,32 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import cloud.orbit.actors.NodeState;
+import cloud.orbit.actors.NodeType;
 import cloud.orbit.concurrent.Task;
 import cloud.orbit.exception.UncheckedException;
 
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.nio.file.Paths;
+import java.util.HashSet;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.TreeMap;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.Executor;
 import java.util.concurrent.ForkJoinTask;
 
+/**
+ * WARNING - JGroupsClusterPeer is currently unsupported (after the clustering algorithm refactors).
```

Comment:
Relevant only to the new clustering algorithm. New clustering algorithm would need to be re-implemented on top of JGroups if we intend to keep JGroupsClusterPeer functional.